.circleci/config.yml

```diff
@@ -222,10 +222,10 @@ jobs:
           command: yarn ckeditor5-dev-ci-circle-disable-auto-cancel-builds
       - run:
           name: Prepare the new version to release
-          command: npm run release:prepare-packages -- --verbose
+          command: yarn release:prepare-packages -- --verbose
       - run:
           name: Publish the packages
-          command: npm run release:publish-packages -- --verbose
+          command: yarn release:publish-packages -- --verbose
```

Comment:
Please revert this change. Yarn and npm do not share a `npm login` session. Hence, we use `npm run` to release stuff instead of `yarn run`.